client/src/components/graphs/OrganizationalChart.tsx

```diff
@@ -189,260 +151,608 @@ const OrganizationalChart = ({
     pageDispatchers
   })
 
-  const canvas = d3.select(canvasRef.current)
-  const link = d3.select(linkRef.current)
-  const node = d3.select(nodeRef.current)
-  const attachmentsEnabled = !Settings.fields.attachment.featureDisabled
-
-  useEffect(() => {
-    data &&
-      setRoot(
-        d3.hierarchy(data.organization, d =>
-          expanded.includes(d.uuid)
-            ? data.organization.descendantOrgs.filter(
-              org => org.parentOrg?.uuid === d.uuid
-            )
-            : null
-        )
-      )
-  }, [data, expanded])
+  if (done) {
+    return result
+  }
+
+  if (!data || !data.organization) {
```

Comment:
Introduce:
```javascript
  const organization = data?.organization
```
here, then use:
```javascript
  if (!organization) {
```
and also replace `data.organization` everywhere below with just `organization.